docs/examples/managed/GoogleDemo.ipynb

```diff
@@ -39,9 +39,11 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.            embed_model=None,
</pre><p>you can use gemini embedding model. we have incorporated Gemini embedding model already</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>